bower.json

```diff
@@ -21,7 +21,8 @@
   "dependencies": {
     "polymer": "Polymer/polymer#^1.0.0",
     "iron-validatable-behavior": "PolymerElements/iron-validatable-behavior#^1.0.0",
-    "iron-form-element-behavior": "PolymerElements/iron-form-element-behavior#^1.0.0"
+    "iron-form-element-behavior": "PolymerElements/iron-form-element-behavior#^1.0.0",
+    "iron-validator-behavior": "PolymerElements/iron-validator-behavior#^1.0.1"
```

Comment:
I think this might need to be a devDependency, since it's only used for testing. WDYT?